x-pack/plugins/apm/public/components/shared/Links/apm/service_overview_link.tsx

```diff
@@ -9,12 +9,16 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import React from 'react';
-import { APMLink, APMLinkExtendProps } from './APMLink';
+import { APMLink, APMLinkExtendProps, useAPMHref } from './APMLink';
 
 interface ServiceOverviewLinkProps extends APMLinkExtendProps {
   serviceName: string;
 }
 
+export function useServiceOverviewHref(serviceName: string) {
+  return useAPMHref(`/services/${serviceName}/overview`);
+}
+
 export function ServiceOverviewLink({
```

Comment:
Is this still needed if we have the hook? I'm all for simplifying our link helpers and having one way to do it might be better.

Reply:
I opened #83196 to look at this.